app/i18n/locales/en.json

```diff
@@ -6,6 +6,7 @@
   "A_new_owner_will_be_assigned_automatically_to__count__room": "A new owner will be assigned automatically to {{count}} room.",
   "A_new_owner_will_be_assigned_automatically_to__count__rooms": "A new owner will be assigned automatically to {{count}} rooms.",
   "accept": "Accept",
+  "Accessibility_statement": "Accessibility statement",
```

Comment:
Can you add i18n for other languages using AI?
It's not perfect, but it's better than not having it translated.